### PR TITLE
Add stetho

### PR DESCRIPTION
**Pull request by devUndef1ned**
Added stetho for LLC.
Init only for DEBUG

### CODE REVIEW

File: client/build.gradle
Comment (samiuelson) [inline client/build.gradle]: It'd be better to declare dependencies like stetho as `debugImplementation`.

**Review by samiuelson — DISMISSED** [review]

File: client/build.gradle
Comment (devUndef1ned) [reply client/build.gradle]: I've never done that, but if I understand correctly with such debugImplementation you have to divide code (where you init stetho and add interceptor) to debug and non debug versions. I covered it via `if (BuildConfig.DEBUG) `condition

Comment (JcMinarro) [reply client/build.gradle]: The problem with that is that we are adding an extra lib to all apps that is using our client.
We shouldn't use it on the final version, that is because we should use it as `debugImplementation`

Comment (zsmb13) [reply client/build.gradle]: You do have to divide the code into debug and release versions for `debugImplementation`, but that way the dependency won't be packaged into release builds at all. With the `BuildConfig.DEBUG` check, it'd be inactive since we never init it, but all of its classes are still present in the output artifact. This both makes the apps built on it larger and it's a potential security issue, if someone finds a way to activate it in the release version.

**Review by samiuelson — APPROVED** [review]
Looks good 👍🏼

File: client/build.gradle
Comment (JcMinarro) [reply client/build.gradle]: That is true, the artifact won't be created with the debug flavor.
If we want to use Stetho, it should be included on the sample app of LLC or wait until we merge the three SDK to be able to work with the `debugImplementation` approach

**Pull request merged**
